specification/storage/resource-manager/Microsoft.Storage/stable/2021-04-01/common.json

```diff
@@ -0,0 +1,137 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Storage",
+    "description": "Storage Resource Provider API Common Types",
+    "version": "2021-04-01"
```

Comment:
The API version 2021-04-01 still not available in any region like canary.
Per @HimanshuChhabra "2021-04-01 ARM manifest rollout is currently in progress and will finish by thursday".
We should not merge this PR before the new API version is available on server.